### PR TITLE
jsonschema: correct obsolete_institutions

### PR DESCRIPTION
**Pull request by kaplun**
Sentry: https://sentry.cern.ch/inspire-sentry/inspire-labs-qa/group/601660/

* Merges obsolete_institutions into related_institutes.

Signed-off-by: Samuele Kaplun <samuele.kaplun@cern.ch>

### CODE REVIEW

File: inspirehep/modules/records/jsonschemas/records/institutions.json [hunk 1]
```diff
@@ -131,20 +131,30 @@
...
-            "description": "HEP affiliation (spires name)",
-            "title": "Old SPIRES control number",
-            "type": "string"
+        "superseded_institutions": {
+            "items": {
+                "additionalProperties": false,
+                "properties": {
+                    "curated_relation": {
+                        "description": "Did a cataloguer proof-checked the recid?",
+                        "title": "The affiliation is curated?",
```
Comment (jacquerie) [inline inspirehep/modules/records/jsonschemas/records/institutions.json]: Can't see the point of these.

File: inspirehep/modules/records/jsonschemas/records/institutions.json [hunk 2]
```diff
@@ -131,20 +131,30 @@
...
+                        "type": "boolean"
+                    },
+                    "record": {
+                        "$ref": "elements/json_reference.json",
+                        "description": "URI for the Institution collection record",
+                        "title": "URI for the Institution collection record"
```
Comment (jacquerie) [inline inspirehep/modules/records/jsonschemas/records/institutions.json]: Also here.

File: inspirehep/modules/records/jsonschemas/records/institutions.json [hunk 3]
```diff
@@ -131,20 +131,30 @@
...
+                    },
+                    "ICN": {
+                        "description": "The affiliation as it appears on the paper",
+                        "title": "Name of institution",
```
Comment (jacquerie) [inline inspirehep/modules/records/jsonschemas/records/institutions.json]: And here.

File: tests/unit/dojson/test_dojson_institutions.py [hunk 4]
```diff
@@ -30,6 +30,41 @@
 from inspirehep.dojson.utils import clean_record
 
 
+def test_superseded_institutions():
```
Comment (jacquerie) [inline tests/unit/dojson/test_dojson_institutions.py]: This test should be called `test_name_from_110__a_t_u_double_x_double_z`. Also it should be closer to the other tests that talk about the `name` rule, a bit lower in the file.

File: inspirehep/dojson/institutions/fields/bd1xx.py [hunk 5]
```diff
@@ -72,9 +72,28 @@ def set_value(key, val):
...
+                'record': get_record_ref(recid, record_type='institutions'),
+                'relation_type': 'superseded',
+            })
+    else:
+        for icn in superseded_icns:
+            related_institutes.append({
+                'curated_relation': False,
+                'name': icn,
+                'relation_type': 'superseded',
+            })
```
Comment (jacquerie) [inline inspirehep/dojson/institutions/fields/bd1xx.py]: :+1:, but we should really abstract this guy in a DoJSON helper local to INSPIRE, since we have been repeating this pattern quite a few times.

**Comment by jacquerie** [comment]
:+1: with some comments.

**Comment by jacquerie** [comment]
Oh, and I'd remove the bullet point about Sentry in the commit message and just add a Sentry: link to the first comment in this PR.

File: inspirehep/modules/records/jsonschemas/records/institutions.json (hunk 1 above)
Comment (kaplun) [inline inspirehep/modules/records/jsonschemas/records/institutions.json]: Was a cut & paste.

Comment (kaplun) [inline inspirehep/modules/records/jsonschemas/records/institutions.json]: Actually `superseded_institutions` was a leftover since I have merged everything with `related_institutes`.

Comment (kaplun) [inline inspirehep/modules/records/jsonschemas/records/institutions.json]: @jacquerie question: `institutes` or `institutions`?

Comment (kaplun) [inline inspirehep/modules/records/jsonschemas/records/institutions.json]: Uh oh! Looks like institutions is connected with mental illness:
http://forum.wordreference.com/threads/institute-or-institution.80638/

Comment (jacquerie) [inline inspirehep/modules/records/jsonschemas/records/institutions.json]: http://www.oxforddictionaries.com/definition/english/institution

Comment (kaplun) [inline inspirehep/modules/records/jsonschemas/records/institutions.json]: Yep! So we use institute and institutions interchangeably. We should stick with one, but this will be the subject of another PR.

File: inspirehep/dojson/institutions/fields/bd1xx.py (hunk 5 above)
Comment (kaplun) [inline inspirehep/dojson/institutions/fields/bd1xx.py]: Mmh. Not sure how to best do it in an elegant way. There are many combination involved...

**Comment by jacquerie** [comment]
LGTM
